src/Resources/config/shop_routing.yml

```
@@ -1,13 +1,13 @@
 sylius_refund_plugin_shop_order_credit_memos_partial:
-    path: /_partial/credit-memos/{orderNumber}/
+    path: /_partial/credit-memos/{orderId}/
```

I'm not sure if 
```suggestion
    path: /_partial/orders/{id}/credit-memos/
```
wouldn't be more obvious. /cc @Zales0123  @GSadee 

This path suggests IMHO only one credit memo.

---

👍